Web/starter_project_barbies/pages/blog/my-blogs/[user-id].tsx

```diff
@@ -0,0 +1,70 @@
+import { NextPage } from "next";
+import { useRouter } from "next/router";
+import { useState, useEffect } from "react";
+import { Blog } from "@/types/blog";
+import BlogCard from "@/components/blog/BlogCard";
+
+
+// Fetch blogs from dummyData.json file 
```

Comment:
[nit] the name of the file is in kebab-case
let's be consistent in the comment too